gnome-initial-setup/pages/summary/gis-summary-page.c

```diff
@@ -39,9 +39,15 @@
 
 #define SERVICE_NAME "gdm-password"
 
+#define CONFIG_SUMMARY_GROUP "page.summary"
+#define CONFIG_SUMMARY_CONTENT_MODE_KEY "content-mode"
+#define HACK_CONTENT_MODE "hack"
```

Comment:
Maybe this ship sailed a long time ago, but this information (is it a hack image?) is already available without having to go via the config file. We check the eos-image-version xattr for several other product-specific tweaks here: https://github.com/endlessm/gnome-initial-setup/blob/a3bdc8506e9f80eb7a6bb5f61edb353185caa695/gnome-initial-setup/gis-driver.c#L190-L203

Reply:
I have used the image attribute before, but since we have been using the conf file, I thought it'd be more flexible to do so. In any case, I didn't notice you had an image check already in g-i-s, so I can change this. Thanks.